autorag/nodes/generator/run.py

```diff
@@ -56,13 +56,18 @@ def run_generator_node(
 	# get average token usage
 	token_usages = list(map(lambda x: x["generated_tokens"].apply(len).mean(), results))
 
+	# make rows to metric_inputs
+	generation_gt = list(map(lambda x: x.tolist(), qa_data["generation_gt"].tolist()))
```

Comment:
Or you can use `to_list()` function from autorag.utils.util to make generation_gt columns to python list recursively